### DISCUSSION TITLE
Bandwidth Throttling with yt-dlp or youtube-playlists?

### DISCUSSION THREAD

**Discussion by madgabz** [smplayer-dev/smplayer#463] [Q&A]
Hi, I'm noticing some throttling when using smplayers EXCELLENT playlist-playback feature for youtube playlists. It seems to be throttling at app. 1 MB/s / ~9 mbit/s. I'm almost sure, it has nothing to do with my computer or network/router throttling, it happens on both cabled and wireless connection, neither is my openwrt router setup to throttle anything the like.
I'm trying to view some big playlists using speed 1.3+ and it seems, the background downloading/caching of video-content cant keep up, and videos start to stutter/pause.
I'm on Manjaro all around here, and might just have to go back to yt-dlp'ing playlists on my NAS, that would be such a waste of opportunity, don't you think?
I'm trying to find out where exactly yt-dlp is setup, but cant seem to dig it out....

**Comment by smplayer-dev** [comment]
Be sure you're using the latest version of yt-dlp. SMPlayer can download it for you using the option "Install / Update Youtube support" in the Help menu.

**Reply by madgabz** [reply]
I was already having latest yt-dlp installed, but used help menu to get latest version, which is 2022.5.18... still, no change, starting then stopping videos still makes background caching use max ~900 KB/s.... 
Thanks for the heads up, was quick :)
:+1:
